variables.tf

```diff
@@ -0,0 +1,112 @@
+variable "kvstores" {
+  type = map(object({
+    autoscaler = optional(object({
+      enabled = optional(bool, true)
+      read_schedule = optional(list(object({
+        schedule     = string
+        min_capacity = number
+        max_capacity = number
+      })), [])
+      write_schedule = optional(list(object({
+        schedule     = string
+        min_capacity = number
+        max_capacity = number
+      })), [])
+    }), {})
+    table = optional(object({
+      billing_mode = optional(string, "PROVISIONED")
+    }), {})
+    redis = optional(object({
+      cpu_size = optional(number, 25),
+      deployment = optional(object({
+        image = optional(object({
+          repository = optional(string, "redis")
+          tag        = optional(string, "7-alpine")
+        }), {})
+        desired_count           = optional(number, 1)
+        maximum_percent         = optional(number, 100)
+        minimum_healthy_percent = optional(number, 0)
+      }), {})
+      enabled             = optional(bool, true),
+      memory_size         = optional(string, "25")
+      service_memory_size = optional(number, 50)
+      service_placement_constraints = optional(set(object({
+        type       = string
+        expression = string
+      })), [])
+    }), {})
+  }))
+  description = "Kvstores to be created"
+  default     = {}
+}
+
+variable "attributes_as_suffix" {
+  type        = bool
+  description = "Attributes passed into the module are put in front of each.key (model name) by default. Setting this to true will put the attributes after the each.key."
+  default     = false
+}
+
+variable "ddb_label_order" {
+  type        = list(string)
+  default     = null
+  description = <<-EOT
+    The order in which the labels (ID elements) appear in the `id`.
+    Defaults to ["namespace", "environment", "stage", "name", "attributes"].
+    You can omit any of the 6 labels ("tenant" is the 6th), but at least one must be present.
+    EOT
+}
```

Comment:
I guess I will change that in a follow-up PR?

Reply:
Let me adjust